build-system/tasks/extension-helpers.js

```diff
@@ -17,7 +17,9 @@
 const colors = require('ansi-colors');
 const debounce = require('debounce');
 const fs = require('fs-extra');
+const globby = require('globby');
 const log = require('fancy-log');
+const pathmod = require('path');
```

Comment:
Another `path` clash rears its head ☹️ 

For consistency with other parts of the codebase, we call this `pathModule` instead of `pathmod`? A future cleanup is forthcoming.

Reply:
removed since we enumerate the paths now